x-pack/plugins/cases/public/components/create/description.test.tsx

```diff
@@ -18,7 +18,7 @@ import { FormTestComponent } from '../../common/test_utils';
 import type { FormSchema } from '@kbn/index-management-plugin/public/shared_imports';
 
 // FLAKY: https://github.com/elastic/kibana/issues/175204
```

Comment:
Do you mind if you connector the issue to the PR to be closed and remove this comment?

Reply:
Sure!